key.go

```diff
@@ -46,31 +46,29 @@ func getApiKeyEnv() (apikey string, err error) {
 
 func getApiKeyFile() (apikey string, err error) {
```

Comment:
Yeah, this is my fault.  I guess it's supposed to mean 'get API key from _some_ file, as opposed to the environment variable (`getApiKeyEnv`) or a specific file (`getApiKeyFromFile`).

Maybe rename these to `keyFromKnownFiles`, `keyFromEnv`, and `keyFromFile` respectively?